two-factor.php

```diff
@@ -139,6 +139,17 @@ function wpcom_vip_enforce_two_factor_plugin() {
 			return $limited;
 		}, 9 );
 
+		// Calcuate two factor authentication support outside map_meta_cap to avoid callback loop
+		// see: https://github.com/Automattic/vip-go-mu-plugins/pull/1445#issuecomment-592124810
+		$is_user_using_two_factor = Two_Factor_Core::is_user_using_two_factor();
```

Comment:
If someone tries to stop 2FA, by doing `remove_action( 'muplugins_loaded', 'wpcom_enable_two_factor_plugin' );`, then this `Two_Factor_Core::...` call is likely to cause a fatal error. It may be worth being defensive and doing a `class_exists()` check anyway.

Reply:
That would be a `_doing_it_wrong`. Plus, there are other pathways in this codebase that would likely also trigger fatals.

Reply:
If that's the case, probably can do away with [this `class_exists` check](https://github.com/Automattic/vip-go-mu-plugins/blob/b8baa5ec0a4bc18aa58990b24eaf74c0aab4735d/two-factor.php#L31) as well, that's what was keeping the previous placement of `Two_Factor_Core` from throwing a fatal